app/scripts/require_config.js

```diff
@@ -13,6 +13,7 @@ require.config({
     mustache: '../bower_components/mustache/mustache',
     stache: '../bower_components/requirejs-mustache/stache',
     chai: '../bower_components/chai/chai',
+    cocktail: '../bower_components/cocktail/Cocktail',
```

Comment:
So awesome, I wrote something like this a while back for a non-backbone app and wanted to see something like this here too.